src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/measure/ActionMeterTest.kt

```diff
@@ -91,11 +86,7 @@ class ActionMeterTest {
             .appendPostMetricHook(DrillDownHook(HardcodedTimeline(entries)))
             .build()
 
-        try {
-            actionMeter.measure(CREATE_ISSUE) { throw Exception("Ignore this exception, it's test-only") }
-        } catch (e: Exception) {
-            logger.info("Ignoring exception", e)
-        }
+        expectTestException { actionMeter.measure(CREATE_ISSUE) { throw TestException() } }
```

Comment:
We can [do even better with AssertJ](https://www.baeldung.com/assertj-exception-assertion#java-8-assertions).

Reply:
Good catch, I didn't suppose it has .hasCauseInstanceOf(..)